deps/Chakra.ICU/Chakra.ICU.Stubdata.vcxproj

```diff
@@ -31,6 +32,12 @@
         $(IcuSourceDirectory)\common
       </AdditionalIncludeDirectories>
     </ClCompile>
+    <Link Condition="'$(ChakraICU)'=='shared'">
+      <SubSystem>Console</SubSystem>
+      <!-- Make Chakra.ICU.Stubdata pretend like its Chakra.ICU.Data for linking purposes -->
+      <OutputFile>$(OutDir)\Chakra.ICU.Data.dll</OutputFile>
+      <ImportLibrary>$(OutDir)\Chakra.ICU.Data.lib</ImportLibrary>
```

Comment:
@curtisman is there any way around this? At runtime, we want Common.dll to auto-load Data.dll, but to create Data.dll we need Common. Thus, Stubdata is built and common links against Stubdata.lib... normally. Except then Common will try to auto-load Stubdata.dll rather than Data.dll, which isn't what we want. I couldn't find a way to work around this, and having the stubdata project output misnamed files like this is what stubdata.vcxproj in upstream ICU does.